<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slot Machine</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Addmoney_Label">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>40</y>
      <width>170</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add money</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RTDB_Button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Return to dashboard</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Add_Button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Money_Line">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Insert desired amount of money here</string>
    </property>
   </widget>
   <widget class="QLabel" name="Error_Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
